internal/stack/shellinit.go

```diff
@@ -23,24 +24,57 @@ var (
 	CACertificateEnv         = elasticPackageEnvPrefix + "CA_CERT"
 )
 
-var shellInitFormat = "export " + ElasticsearchHostEnv + "=%s\n" +
-	"export " + ElasticsearchUsernameEnv + "=%s\n" +
-	"export " + ElasticsearchPasswordEnv + "=%s\n" +
-	"export " + KibanaHostEnv + "=%s\n" +
-	"export " + CACertificateEnv + "=%s"
-
 // ShellInit method exposes environment variables that can be used for testing purposes.
-func ShellInit(elasticStackProfile *profile.Profile) (string, error) {
+func ShellInit(elasticStackProfile *profile.Profile, st string) (string, error) {
 	config, err := StackInitConfig(elasticStackProfile)
 	if err != nil {
 		return "", nil
 	}
 
-	return fmt.Sprintf(shellInitFormat,
-		config.ElasticsearchHostPort,
-		config.ElasticsearchUsername,
-		config.ElasticsearchPassword,
-		config.KibanaHostPort,
-		config.CACertificatePath,
+	// NOTE: to add new env vars, the template need to be adjusted
+	return fmt.Sprintf(initTemplate(st),
+		ElasticsearchHostEnv, config.ElasticsearchHostPort,
+		ElasticsearchUsernameEnv, config.ElasticsearchUsername,
+		ElasticsearchPasswordEnv, config.ElasticsearchPassword,
+		KibanaHostEnv, config.KibanaHostPort,
+		CACertificateEnv, config.CACertificatePath,
 	), nil
 }
+
+const (
+	// shell init code for POSIX compliant shells.
+	// IEEE POSIX Shell and Tools portion of the IEEE POSIX specification (IEEE Standard 1003.1)
+	posixTemplate = `export %s=%s
+	export %s=%s
+	export %s=%s
+	export %s=%s
+	export %s=%s
```

Comment:
This is adding spaces before each `export` right? Can you check the output of `elastic-package stack shellinit`?

Reply:
fixed in [`7a98775` (#921)](https://github.com/elastic/elastic-package/pull/921/commits/7a9877558c9dc80c47fdfd3bb15fc1a481f070d3)